README.md

```diff
@@ -51,6 +51,8 @@ features = ["extension-module"]
 **`src/lib.rs`**
 
 ```rust
+extern crate pyo3;
```

Comment:
Please add a note in the readme that `extern crate pyo3;` is not required when using the 2018 edition

Reply:
Done